docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -96,20 +96,20 @@ To see how these features actually get used in the runtime code, we can open the
 include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 use rstd::prelude::*;
-//--snip--
```

Comment:
Definitely not a blocker, but why prefer this format?

I think we chose `//--snip--` following in the footsteps of the rust book eg https://doc.rust-lang.org/book/ch19-04-advanced-types.html#the-never-type-that-never-returns

Reply:
You are right. I just did it because it was clear to do it this way after comments. Like in this example:
```rust
// These time units are defined in number of blocks.
   /* --snip-- */
```